registrations/models.py

```diff
@@ -49,6 +49,7 @@ class Registration(models.Model):
     REG_TYPE_CHOICES = (
         ('momconnect_prebirth', "MomConnect pregnancy registration"),
         ('momconnect_postbirth', "MomConnect baby registration"),
+        ('whatsapp_prebirth', "WhatsApp MomConnect pregnancy registration"),
```

Comment:
I think we might need a migration for something like this? I'm not sure why, since it doesn't change anything database side, but from what I remember django wants a migration for a change like this.